src/components/single-page/single-page-content.tsx

```diff
@@ -17,39 +15,22 @@ interface IProps {
 
 export const SinglePageContent: React.FC<IProps> = (props) => {
   const { activity, teacherEditionMode, pluginsLoaded } = props;
```

Comment:
I am also seeing TSLint information in github saying that we don't need teacherEditionMode or pluginsLoaded.